documentation/src/main/resources/theme/default-theme.yml

```diff
@@ -0,0 +1,287 @@
+font:
```

Comment:
Is this imposed by Quarkus? Shouldn't this be imported from some URL rather than stored in-tree?

Reply:
That's a problem because we are the first ones (similar to Netty EvenGroup) so I doubt that there is a way to do it.
I did ask a question in the quarkus ticket:
https://github.com/quarkusio/quarkus/issues/7876#issuecomment-626725515

Reply:
Ok I think it's acceptable for now. Maybe we can open a follow-up Jira ticket and repeat your question there.

Reply:
Ok, created https://datastax-oss.atlassian.net/browse/JAVA-2771